web_widget_image_download/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===========================
+Web Widget - Image Download
+===========================
+
+This module was written to extend the functionality of the image widget and allows to download it.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to the section `Contacts`.
+#. Click on a contact.
+#. Edit the contact.
+#. Click the `Download` button (between `Edit` and `Clear`).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/8.0
+
+Known Issues / Roadmap
+======================
+
+* In order to work correctly, this widget has to detect image type, the server should include this information in the `Content-Type` header. Right now, odoo is not doing so, but a fix has been `proposed <https://github.com/odoo/odoo/pull/12918>`_.
+* For some unknown reason, the widget does not work in the `Preferences` view, because odoo is not rendering the **QWeb** template.
```

Comment:
@Yajo you should remove this from the **Known Issues**, as it is solved by your last commit :+1: the same happens in the PR for the v8.0.